src/libcore/macros.rs

```diff
@@ -349,6 +349,27 @@ macro_rules! try {
 /// write!(&mut v, "s = {:?}", s).unwrap(); // uses io::Write::write_fmt
 /// assert_eq!(v, b"s = \"abc 123\"");
 /// ```
+///
+/// Note : This macro can be used in no_std setups as well
```

Comment:
can you remove the space before the `:` please?

and `no_std` needs backticks, here and on the next line